modules/core/src/main/java/org/apache/ignite/configuration/ThinClientConfiguration.java

```diff
@@ -103,22 +112,45 @@ public ThinClientConfiguration setMaxActiveComputeTasksPerConnection(int maxActi
 
     /**
      * @return If {@code true} sends a server exception stack to the client side.
+     * @deprecated please now use new method getServerExceptionStackTraceToClient().
      */
+    @Deprecated
     public boolean sendServerExceptionStackTraceToClient() {
         return sendServerExcStackTraceToClient;
     }
 
     /**
      * @param sendServerExcStackTraceToClient If {@code true} sends a server exception stack to the client side.
      * @return {@code this} for chaining.
+     * @deprecated please now use new method setServerExceptionStackTraceToClient().
      */
+    @Deprecated
     public ThinClientConfiguration sendServerExceptionStackTraceToClient(boolean sendServerExcStackTraceToClient) {
         this.sendServerExcStackTraceToClient = sendServerExcStackTraceToClient;
 
         return this;
     }
 
-    /** {@inheritDoc} */
+    /**
+     * @return If {@code true} gets a server exception stack that is to be sent to the client side.
+     */
+    public boolean getServerExceptionStackTraceToClient() {
+        return serverExcStackTraceToClient;
+    }
+
+    /**
+     * @param serverExcStackTraceToClient if {@code true} sets a server exception stack that is to be sent to the client side.
+     * @return {@code this} for chaining.
+     */
+    public ThinClientConfiguration setServerExceptionStackTraceToClient(boolean serverExcStackTraceToClient) {
```

Comment:
Is not clear from the method's name what exactly this method set (javadoc also confusing). How about `setServerToClientExceptionStackTraceSending(boolean sendStackTrace)`?

Reply:
Agree, I have changed the method name as per your recommendation above.
Java doc then is "@param sendStackTrace if {@code true} sets a server to client exception stack trace sending." 
Do you agree?

Reply:
Javadoc for this parameter of the old method looks better to me, but it's up to you. 
Also, the first word of the javadoc should start with the upper case letter.